### ISSUE TITLE
Source Maps not Working

### ISSUE THREAD

**Issue by Lalem001** [jlmakes/karma-rollup-preprocessor#36]
Picking up after #33, which was awaiting reproduction, I have created a repo that reproduces the issue. Been 5 days since I posted in that issue with no apparent attention. So starting a new issue.

Basically when a test error occurs, the reported error line is incorrect.

https://github.com/Lalem001/karma-rollup-preprocessor-issue

**Comment by jlmakes** [comment]
I wouldn't suspect the `chokidar` file watcher, which means there's only about [40 lines of code](https://github.com/jlmakes/karma-rollup-preprocessor/blob/master/lib/index.js#L7-L46) that would contain the bug. Maybe something with the `cache`?

I’ll take a closer look soon, thanks @Lalem001
